cvat_utils/xml_utils.py

```diff
@@ -0,0 +1,70 @@
+from bs4 import BeautifulSoup
+
+def baseline_info_xml(cvat_soup):
+    xml_root = "<annotations></annotations>"
+    xml_soup = BeautifulSoup(xml_root, 'xml')
+
+    xml_soup.annotations.append(cvat_soup.find("version"))
+    xml_soup.annotations.append(cvat_soup.find("meta"))
+
+    return xml_soup
+
+def f2b_to_xml(all_flatten_dets, smallie_annots, f2b_merged_dicts, xml_soup):
+    smallie_annots_filtered = filter_smallie_annots(smallie_annots, f2b_merged_dicts)
+
+    total_annots = 0
+    for i, flatten_dets in enumerate(all_flatten_dets):
+        xml_img = xml_soup.find("image", {"id": i})
+        for flatten_det in flatten_dets:
+            xml_info = smallie_annots_filtered[total_annots]
+
+            xml_det = xml_soup.new_tag("box", occluded=xml_info.get("occluded"), source=xml_info.get("source"), z_order=xml_info.get("z_order"))
```

Comment:
Need to handle cases when some of the attributes not available/present. In my case, I had a cvat xml that does not have "z_order" attribute in the annotation. The resulting xml is not a valid xml, with lines like this: ` <box label="person" occluded="0" source="manual" xbr="2245.73" xtl="2207.51" ybr="987.5" ytl="943.17" z_order/>`